### PR TITLE
feat(lib): now c:get_result() return the count of captures

### PR DESCRIPTION
**Pull request by chronolaw**
Related PR: https://github.com/Kong/kong/pull/13024

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ If an error occurred, `nil` and a string describing the error will be returned.
 
 ### get\_result
 
-**syntax:** *uuid, matched_value, captures = c:get_result(matched_field)*
```
Comment (ADD-SP) [inline README.md:194]: We'd better explain what the `captures_len` means.

**Comment by ADD-SP** [comment]
Rebased on master

File: README.md (hunk 1 above)
Comment (chronolaw) [reply README.md:195]: added.

**Review by nowNick — APPROVED** [review]

**Comment by chronolaw** [comment]
I think that we already have a good enough solution in kong gateway, so this PR will be closed.

**Pull request closed without merging**